who-we-are.md

```diff
@@ -32,26 +28,45 @@ We promote a company culture that appreciates collaborative professional growth.
-
-Career growth is facilitated and tracked via the Buddy System. Individuals meet routinely with a buddy of their choice (not all team members are buddies, but every team member has a buddy) to discuss career goals, track goal progress, and discuss feedback from the rest of the team. Our flat hierarchy means that no one has a manager, and the Buddy System is no different: we do not review our teammates, we advocate for them. All Buddy System documents are transparent and [available on the shared Drive](https://b2io.slack.com/archives/C904HLD5L/p1516982662000944).
+In addition we also provide access to online training opportunities and sponsor employee attendance at various technical conferences. We encourage community involvement through technical meetups and open source contributions.
 
+Career growth is facilitated and tracked. Individuals meet routinely with their supervisor to discuss career goals, track goal progress, and discuss feedback from the rest of the team. Our flat hierarchy means that no one has a direct manager. we do not review our teammates, We advocate for them.
 
 ### Honest and Open
 
 We operate in a very honest and collaborative manner. Everyone has the opportunity to voice their opinion on all company decisions - from the type of coffee we purchase to the appropriate salary for a potential hire. This sharing of information allows all employees to be equally informed so that we can make the best collaborative decisions as a company. With that, all employees have visibility into the company’s financials - including internally transparent salaries.
 
-
 #### Transparency
 
 Base Two is an internally transparent company. We make the majority of decisions regarding the company as a group and all employee's opinions are valued, sought after, and taken into consideration. All employees have access to the organization's financial information, including the [Base Two Salaries](https://b2io.slack.com/archives/C904HLD5L/p1516982712000070) document, which shows the team's compensation levels. We [meet once a week](./day-to-day.md#weekly-meeting) as a team to discuss a broad range of company-related topics.
 
-
 #### Cost of Living Adjustments
 
 Cost of living adjustments are applied each January based on the previous year's [Social Security COLA](https://www.ssa.gov/oact/cola/colaseries.html). The adjustment applies to all full-time employees as of 12/31 of the previous year.
 
-
 #### Raises
 
 If a team member believes that they (or any other member of the team) deserve a raise, then it should be brought up for discussion either in Buddy meetings, small groups, the Weekly Meeting, or whichever setting feels most comfortable. These discussions have tended to be informal in the past, and focus on the value and responsibilities that the team member provides.
+
+#### Conferences
```

Comment:
Add the conferences you've attended while at B2 here! (I went to DrupalGC like a month before I started here, so I'm counting that.)